app/src/androidTest/java/com/github/polybooks/LoginGoogleTest.kt

```diff
@@ -38,54 +38,6 @@ class LoginGoogleTest {
 
     @Test
     fun signInButtonGoogle() {
-        onView(withId(com.github.polybooks.R.id.sign_in_button)).perform(scrollTo(), click())
-    }
-
-    @Test
-    fun signOutButtonGoogle() {
-        onView(withId(com.github.polybooks.R.id.register_button)).perform(scrollTo(), click())
-    }
-
-    @Test
-    fun navBarSales() {
-        onView(withId(com.github.polybooks.R.id.sales)).perform(click())
-        Intents.intended(IntentMatchers.hasComponent(FilteringSalesActivity::class.java.name))
-    }
-
-    @Test
-    fun navBarProfile() {
-        onView(withId(com.github.polybooks.R.id.user_profile)).perform(click())
-        onView(withId(com.github.polybooks.R.id.sign_in_button)).perform(scrollTo()).check(ViewAssertions.matches(isDisplayed()))
-    }
-
-    @Test
-    fun navBarBooks() {
-        onView(withId(com.github.polybooks.R.id.books)).perform(click())
-        Intents.intended(IntentMatchers.hasComponent(FilteringBooksActivity::class.java.name))
-    }
-
-    @Test
-    fun navBarDefault() {
-        onView(withId(com.github.polybooks.R.id.default_selected)).check(
```

Comment:
Is there a reason to not test the nav bar anymore?

Reply:
I had to delete the nav bar from the login and register activity for now to facilitate the tests, but it will be added in a different PR

Reply:
okay!